### ISSUE TITLE
how to using nvdla_runtime? how can I get the prediction result?

### ISSUE THREAD

**Issue by cgl-nudt** [nvdla/sw#92]
I use the command "./nvdla_runtime --softmax --loadable basic.nvdla --image one.pgm", why the result is only 
`Work Found!
Work Done
Shutdown signal received, exiting
Test pass
`
how can I get the prediction result. I try to modify the file of "runtime.cpp" and "emulator.cpp", but the result is also the 
`Work Found!
Work Done
Shutdown signal received, exiting
Test pass`

**Comment by JunningWu** [comment]
which neural network model did you use? Lenet-5? please reference the output.dim file, It will give ten predicted results.

**Comment by cgl-nudt** [comment]
@JunningWu hi！I use the Lnet-5 and I find a output.dimg file, but  I don't know how to use it.

**Comment by XiaochaoGONG** [comment]
with `--rawdump` you will get the probability of the result.
